css/style.css

```diff
@@ -6,3 +6,107 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+
+
+ 
+
+ body{
+    background-image: url(../images/celeste.gif);
+    background-repeat: no-repeat;
+    background-size:contain
+}
+
+.main-text{
+    font-size: 130pt;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-weight:bolder;
+    color: whitesmoke;
+    text-align: right;
+    padding-right: 50px;
+    margin-top: 60px;
+    margin-bottom: -60px;
+}
+
+h2{
+    color: azure;
+    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    font-size: 40pt;
+    text-align: right;
+    padding-right: 160px;
+    margin-bottom: 90px;
+}
+.first-text{
+    color: aqua;
+    text-align: right;
+    padding-right: 260px;
+    font-size: 20pt;
+}
+
+.video{
+    position: relative;
+    left: 1100px;
```

Comment:
When the HTML renders, there is a large blank section to the right of the viewport being rendered. I believe it is this CSS property that is creating that blank space. Consider different ways of positioning the video element on the page.